src/main/java/org/carlmontrobotics/commands/Intake.java

```diff
@@ -2,11 +2,14 @@
 
 import static org.carlmontrobotics.Constants.Effectorc.*;
 
+import org.carlmontrobotics.Constants;
+
 import org.carlmontrobotics.subsystems.IntakeShooter;
+import org.carlmontrobotics.subsystems.Led;
 
 import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj2.command.Command;
-
+import edu.wpi.first.wpilibj2.command.Subsystem;
```

Comment:
None of the additional imports should be needed because nothing else changed in this file.